torch/_decomp/decompositions.py

```diff
@@ -1136,6 +1136,19 @@ def addmm(self: Tensor, mat1: Tensor, mat2: Tensor, beta: int = 1, alpha: int =
     return out + beta * self
 
 
+@register_decomposition(aten.addmv)
+@out_wrapper()
+@pw_cast_for_opmath
+def addmv(self: Tensor, mat1: Tensor, vec: Tensor, beta: int = 1, alpha: int = 1):
```

Comment:
is it correct to annotate here beta/alpha always as int?

Reply:
Hi, seems to be in here as well: https://github.com/pytorch/pytorch/pull/76621/files#diff-d25ad317caf41c3fdf1de3bcfc730102c9e4a3e68639559017ae45cb0bc26911R741

It didn't seem right but I thought it was some convention.. Should I remove this as well?